blog/_posts/2021-11-15-k8s-custom-deploy.md

```diff
@@ -0,0 +1,106 @@
+---
+slug: "k8s-custom-deploy"
+date: 2021-11-18
+author: milas
+layout: blog
+title: "Deploy All The Things Even If They Aren't YAML"
+image: "/assets/images/k8s-custom-deploy/title.jpg"
+description: "Integrate Tilt with custom Kubernetes deployment scripts & tools"
+tags:
+  - kubernetes
+  - tilt
+  - helm
+---
+
+If you've used Tilt, chances are you are well acquainted with the [`k8s_yaml`][api-k8s_yaml] and [`k8s_resource`][api-k8s_resource] Tiltfile functions.
+The [`k8s_yaml`][api-k8s_yaml] function deploys entities to Kubernetes from either a file or string.
+The [`k8s_resource`][api-k8s_resource] function configures port forwards and other behaviors for the deployed entities.
+
```

Comment:
I feel we're missing the problem statement. Maybe add a sentence like "But sometimes, a kubernetes manifest is not enough. Maybe you want something to manage the entire lifecycle of multiple k8s resources that all belong together, like Helm"

Reply:
We should also mention early in this blogpost that even though we use Helm as an example, this is applicable to all kinds of k8s deployment patterns